be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/LaoHandler.scala

```diff
@@ -39,9 +41,17 @@ case object LaoHandler extends MessageHandler {
         val data: CreateLao = message.decodedData.get.asInstanceOf[CreateLao]
         // we are using the lao id instead of the message_id at lao creation
         val channel: Channel = Channel(s"${Channel.rootChannelPrefix}${data.id}")
-
-        val ask: Future[GraphMessage] = (dbActor ? DbActor.Write(channel, message)).map {
-          case DbActorWriteAck => Left(rpcMessage)
+        val laoData: LaoData = LaoData(message.sender, List(message.sender)) //so that the operations the owner does next are authorized
```

Comment:
These lines basically add the one sending CreateLao as the owner and as a Roll Call participant (this could maybe be removed, or do we consider that the owner should automatically be present at all Roll Calls?), in order to later on, when I implement the verifiers to check the message sender's identity, allow the owner to directly send messages to the server, but I'll update the comment since it is confusing